simulation/adx_gyro_accelerometer/publish.gradle

```diff
@@ -0,0 +1,75 @@
+apply plugin: 'maven-publish'
+apply plugin: 'edu.wpi.first.wpilib.versioning.WPILibVersioningPlugin'
+
+if (!hasProperty('releaseType')) {
+    WPILibVersion {
+        releaseType = 'dev'
+    }
+}
+
+def pubVersion = ''
+if (project.hasProperty("publishVersion")) {
+    pubVersion = project.publishVersion
+} else {
+    pubVersion = WPILibVersion.version
+}
+
+def baseArtifactId = 'halsim-adx_gyro_accelerometer'
+def artifactGroupId = 'edu.wpi.first.halsim'
+
+def outputsFolder = file("$project.buildDir/outputs")
+
+task cppHeadersZip(type: Zip) {
+    destinationDir = outputsFolder
+    baseName = 'halsim-adx_gyro_accelerometer'
+    classifier = "headers"
+
+    from(licenseFile) {
+        into '/'
+    }
+
+    from('src/main/native/include') {
+        into '/'
+    }
+}
+
+task cppSourcesZip(type: Zip) {
+    destinationDir = outputsFolder
+    baseName = 'halsim-adx_gyro_accelerometer'
+    classifier = "sources"
+
+    from(licenseFile) {
+        into '/'
+    }
+
+    from('src/main/native/cpp') {
+        into '/'
+    }
+}
+
+model {
+    publishing {
+        def libSpec
+        $.components.each {
+            if (it in NativeLibrarySpec) {
+                $.binaries.each {
+                    if (it in SharedLibraryBinarySpec) {
+                        libSpec = it.sharedLibraryFile
```

Comment:
This really should be a zip in the correct format, especially if we want to be able to statically link a program to it. See the example in wpilibc for how to do this. It's just a function called to create the zip.

Reply:
I basically copied this from halsimprint.  

What artifact are you talking about in wpilibc, wpilibCTaskList or allTask?  One is a nice simple function call, and the other is a more verbose version of what I copied

Reply:
halsimprint got merged wrong, and I want to change it.

The wpilibCTaskList one the one I was thinking of. With a call like that, it will make sure everything gets zipped into the proper formats.

Reply:
@pjreiniger I PR'd our pattern to fix HALSim_Print. You can borrow off of it if you want to see how publishing should look.

https://github.com/wpilibsuite/allwpilib/pull/721/files